test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -104,6 +106,22 @@ var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: C
 			doNotScaleOnBusySidecar("rs", e2eautoscaling.KindReplicaSet, true, f)
 		})
 	})
+
+	ginkgo.Describe("CustomResourceDefinition", func() {
+		ginkgo.It("Should scale with a CRD targetRef", func() {
+			scaleTest := &HPAScaleTest{
+				initPods:                    1,
+				totalInitialCPUUsage:        150,
+				perPodCPURequest:            200,
+				targetCPUUtilizationPercent: 50,
+				minPods:                     1,
+				maxPods:                     2,
+				firstScale:                  2,
+				targetRef:                   e2eautoscaling.CustomCRDTargetRef(),
+			}
+			scaleTest.run("crd-light", e2eautoscaling.KindCRD, f)
```

Comment:
what is this exercising, if the targetref was pointing at something unresolveable (a non-group-qualified CRD reference)?

Reply:
This should be pointing to a valid CRD right now.